bot/admin/admin.py

```diff
@@ -133,6 +133,85 @@ async def embed_available_cogs(self, ctx: commands.Context):
 
         await ch_bot.send(embed=embed)
 
+    @cmd_for_bot_stuff.command(name='load', hidden=True)
+    @command_log
+    async def load_extension(self, ctx: commands.Context, extn_name: str):
+        """Command handler for the `bot` subcommand `load`.
+
+        Loads an extension (Cog) with the specified name into the bot.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+            extn_name (str): The name of the extension (Cog).
+        """
+        extn_name = extn_name.capitalize()
+
+        if "Cog" and "cog" not in extn_name:
```

Comment:
Also this check is completely wrong I think. IMO it evaluates: 
`"Cog" == True && "cog" not in extn_name`

So the first "Cog" is only evaluated if it is true, and a non-empty string is always true.

Reply:
Btw. thats the stuff you should test in a test. If it correctly evaluates and adds the strings (therefor it also would fit fine in a private function)

Reply:
You're completely right. It seems that I didn't have enough caffeine this morning and my brain probably thought that this makes sense in the way you read this. 😂